front/src/modules/pathfinding/components/Itinerary/DisplayItinerary/v2/ViasV2.tsx

```diff
@@ -61,7 +63,7 @@ const ViasV2 = ({ zoomToFeaturePoint, shouldManageStopDuration }: DisplayViasV2P
                       >
                         <small className="font-weight-bold text-muted mr-1">{index + 1}</small>
                         <small data-testid="via-dropped-name" className="mr-1 text-nowrap">
-                          {`${via.name || `KM ${via.positionOnPath && (Math.round(via.positionOnPath) / 1000000).toFixed(3)}`}`}
+                          {`${via.name || (via.positionOnPath && `KM ${(Math.round(via.positionOnPath) / 1000000).toFixed(3)}`) || `${t('requestedPoint')} ${index + 1}`}`}
```

Comment:
Maybe we can add the via counter inside the trad with a `{{ count }}` ?

Reply:
As seen with @thibautsailly, it will now be "distance indisponible" / "unavailable distance" instead of "point demandé" / "requested point"